django_dramatiq/management/commands/rundramatiq.py

```diff
@@ -174,5 +175,9 @@ def _get_submodules(self, package):
     def _resolve_executable(self, exec_name):
         bin_dir = os.path.dirname(sys.executable)
         if bin_dir:
-            return os.path.join(bin_dir, exec_name)
+            # Return first matching path found in directories
+            dirs = [bin_dir, os.path.join(bin_dir, 'Scripts')]
+            for d in dirs:
+                for path in glob.iglob(os.path.join(d, exec_name)):
```

Comment:
Is the glob necessary or could something like:

```python
for d in dirs:
  exec_path = os.path.join(d, exec_name)
  if os.exists(exec_path):
    return exec_path
```

work instead?

Reply:
Ok. I will update it.